wasm2c/wasm-rt-impl.c

```diff
@@ -42,8 +42,6 @@
 static bool g_signal_handler_installed = false;
 #ifdef _WIN32
 static void* g_sig_handler_handle = 0;
-#else
-static char* g_alt_stack = 0;
 #endif
```

Comment:
Maybe too much of a pain but removing g_alt_stack now seems like a separable change, along with `SS_DISABLE` stuff?

Reply:
Okay, let me try to split this out into a new PR... (And maybe we should keep track of it in a thread-local just for safety anyway.)

Reply:
Done.